lib/cluster.js

```diff
@@ -127,13 +127,13 @@ function RoundRobinHandle(key, address, port, addressType, fd) {
     this.server.listen(port, address);
   else
     this.server.listen(address);  // UNIX socket path.
-
-  this.server.once('listening', () => {
```

Comment:
isn't this a convoluted way of writing

```
this.server.once('listening', function onceListening() {
```

Why are we using es6 short forms here if they are not fit for purpose? I'm also quite surprised that assigning a Function to a const magically makes it get named, you are certain that this is achieving the intended goal?